### PR TITLE
Add ITK-SNAP 3.6.0

### PR DESCRIPTION
**Pull request by fepegar**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

The style check failed because the download URL doesn't match the main page. Can you guys please verify that?

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/itk-snap.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'itk-snap' do
+  version '3.4.0'
+  sha256 'f1b6d98db85e2f727402860baf6a7f9ff39e53aa4d27bc187ecf863cc5b86d59'
+
+  url 'http://www.nitrc.org/frs/downloadlink.php/7960'
```
Comment (commitay) [inline Casks/itk-snap.rb]: This redirects to a sourceforge download, please change it to a [direct download](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls) and add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) and a verified comment ([example](https://github.com/caskroom/homebrew-cask/blob/f303a0b431fe1a991464e20fcfec5e010017e91d/Casks/visual-studio-code.rb#L5))

**Review by commitay — APPROVED** [review]

**Comment by fepegar** [comment]
I was waiting to read the doc you gave me so that I could answer.

I thought 3.4.0 should be here and 3.6.0 in `homebrew-versions`. Anyway I don't think many people use the stable version. Thanks for reviewing.

**Comment by commitay** [comment]
> I was waiting to read the doc you gave me so that I could answer.

I checked the `appcast` and realised that following the example in the doc wouldn't work in this case.

> I thought 3.4.0 should be here and 3.6.0 in homebrew-versions.

`3.6.0` looks like it is the current version. 
> http://www.itksnap.org/pmwiki/pmwiki.php?n=Downloads.SNAP3

**Comment by fepegar** [comment]
> `3.6.0` looks like it is the current version.

In the [downloads page](http://www.itksnap.org/pmwiki/pmwiki.php?n=Downloads.SNAP3) there are three versions:
* Latest
* Stable
* Nightly

Since the *stable* seems old and the *latest* seems stable, I guess what makes sense is adding the *latest* here and the *nightly* in `versions`. What do you think?

**Comment by commitay** [comment]
Sorry, you're right, I missed the stable version. I followed the latest version on the homepage and the sourceforge page.

> I guess what makes sense is adding the latest here and the nightly in versions

If a nightly cask would be of use to you please feel free to submit a PR to `caskroom/versions` to add it.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**